README.md

```diff
@@ -78,7 +78,6 @@ The rest of the components in the table are pure upstream OpenTelemetry componen
 |                    [otlp][otlpreceiver]                    |                                                              |                                        |                                              |
 |               [podman_stats][podmanreceiver]               |                                                              |                                        |                                              |
 |              [postgresql][postgresqlreceiver]              |                                                              |                                        |                                              |
-|         [prometheus_exec][prometheusexecreceiver]          |                                                              |                                        |                                              |
```

Comment:
It was deprecated, not removed yet:

https://github.com/open-telemetry/opentelemetry-collector-contrib/tree/main/receiver/prometheusexecreceiver

Reply:
It's removed from the contrib distribution. https://github.com/open-telemetry/opentelemetry-collector-releases/pull/152

My understanding is we include all the components that are included in otelcol and otelcol-contrib distributions, not necessarily all the components that are included in the core or contrib source repository.